### ISSUE TITLE
IpcSender should consistently be Sync or !Sync.

### ISSUE THREAD

**Issue by Ms2ger** [servo/ipc-channel#135]
It seems like it's `Sync` on linux and `!Sync` on windows right now.

**Comment by emilio** [comment]
yeah, that's because on windows we're using mpsc, which is out of our control. Worth waiting to #108 to land?

**Comment by antrik** [comment]
@Ms2ger it should always be `!Sync` after https://github.com/servo/ipc-channel/pull/115 -- it just hasn't landed in Servo yet. (Because of unrelated regressions in other commits...)

I'm am confused though: I did believe the version currently used in Servo should actually be `Sync` everywhere?... Can you please elaborate why you think that's not the case?

**Comment by Ms2ger** [comment]
I didn't double-check, but I thought mpsc senders were not Sync? Anyway, we should land that.
